combat-trainer.lic

```diff
@@ -355,8 +354,17 @@ class LootProcess
   end
 
   def should_perform_ritual(prepare_consume)
-    !prepare_consume &&
-      ((@necro_train_first_aid && DRSkill.getxp('First Aid') < 32) || DRSkill.getxp('Thanatology') < 32)
+   !prepare_consume &&
```

Comment:
Would this be clearer if you break it up into multiple `return`s? Similar to line 364.